Tutorials/Submitting_analysis_jobs_to_gadi.ipynb

```diff
@@ -11,7 +11,7 @@
     "\n",
```

Comment:
<p>let's drop the mention to <code>cosima_cookbook</code> since it's deprecated? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/380/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Ok done